### ISSUE TITLE
[Discuss] Escape key from My Account, LiveChat, Admin, etc. areas

### ISSUE THREAD

**Issue by acamarata** [RocketChat/Rocket.Chat#1872]
Would it be a good thing to have the escape key exit out of the Admin area as an easy and quick way to get back to the last chat, channel, etc. you were active in?

**Comment by blyry** [comment]
I came here looking for this exact issue. I keep rocket.chat on a half screen with snap, which is small enough to automatically collapse the left sidebar with my name, favorites, channels etc. 

It drives me INSANE that 

1) to expand the sidebar you click in the top left, but clicking again in the top left does NOT collapse the sidebar. It makes expanding and collapsing to quick see what group there is a notification on a multistep process.

2) I would expect that to collapse the sidebar again I could either click off of it or click on a collapse button somewhere, having to find the room i'm currently in and click on it to collapse is annoying

tl;dr; Either modify the UI so there is an explicit collapse button, make the responsiveness tweakable (so on a half screen the sidebar can stay always open without overflowing chat content off to the right, or at the very least let the escape key collapse the left sidebar.

**Comment by blyry** [comment]
see #1826 for not directly related, but sidebar-related discussions.

**Comment by MartinSchoeler** [comment]
@acamarata Today you can exit most screens with the escape key.
